docker-compose-test.yml

```diff
@@ -0,0 +1,133 @@
+# todo: simplify or load minimum test info needed from local files 
+# for teardown and startup and to remove volumes
+
+services:
+
+  httpd:
+    build: apache
+    links:
+    - "tomcat-pn"
+    - "xsugar"
+    - "fuseki"
+    - "solr"
+    - "sosol"
+    volumes:
+    - repo:/srv
+    ports:
+    - "8000:80"
+    depends_on:
+    - xsugar
+    - tomcat-pn
+    - solr
+    - fuseki
+    - sosol
+  sosol:
+    build: sosol
+    ports:
+    - "8080:8080"
+    volumes:
+    # waiting for /srv/data/papyri.info/lockfiles/repo_clone/canonical_cloned.lock
+    - ./selenium-ruby/shim/lockfiles/canonical_cloned.lock:/srv/data/papyri.info/lockfiles/repo_clone/canonical_cloned.lock #add shim
+    # - repo:/srv
+    - ./sosol-compose:/opt/sosol-compose
+    command: /opt/sosol-compose/migrate-and-copy.sh
+    links:
+    - "xsugar"
+    - "postgres"
+    depends_on:
+    - postgres
+    - repo_clone
+    - xsugar
+  repo_clone:
+    build: repo_clone
+  xsugar:
+    build: xsugar
+  navigator:
+    build: navigator-compose
+    depends_on:
+    - repo_clone
+    - fuseki
+    links:
+    - "fuseki"
+  fuseki:
+    # built from https://github.com/dcthree/docker-fuseki
+    image: ryanfb/fuseki
+    # volumes:
+    # - repo:/srv
+    # - fuseki_data:/data
+    cpu_count: 2
+    environment:
+    - OPTS=--port=8090 --tdb2 --loc=/data --update /pi
+    - JVM_ARGS=-Xmx8192M
+    - JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8
+    ports:
+    - "8090:8090"
+  postgres:
+    image: postgres:13
+    environment:
+    - POSTGRES_USER=papyri
+    - POSTGRES_HOST_AUTH_METHOD=trust
+  solr:
+    build: solr
+    ports:
+    - "8983:8983"
+    # volumes:
+    # - solr_data:/opt/solr/server/solr
+    environment:
+    - SOLR_JAVA_MEM=-Xms1G -Xmx8G
+    depends_on:
+    - navigator
+  tomcat-pn:
+    build: tomcat-pn
+    links:
+    - "postgres"
+    - "fuseki"
+    volumes:
+    # waiting for /srv/data/papyri.info/lockfiles/navigator/mapping_done.lock
+    # - ./selenium-ruby/shim/lockfiles/mapping_done.lock:/srv/data/papyri.info/lockfiles/navigator/mapping_done.lock #add shim
+    - repo:/srv
```

Comment:
Last remaining volume where tomcat pulls the .war files. The expectation for running these tests currently is that they already exist. More to come here I'm sure as upstream changes are made